.env.example

```diff
@@ -0,0 +1,3 @@
+# apm install wordpress-develop
```

Comment:
Since this is a public repo, we shouldn't have references to internal tooling. The environment variable lines are generic enough that I'm fine with keeping them.
```suggestion
# Ensure wordpress-develop is installed and available locally.
```